src/main.rs

```diff
@@ -651,6 +644,8 @@ enum ProgramError {
     IoError(io::Error),
     /// Error while processing files for walking.
     IgnoreError(ignore::Error),
+    /// The givon query failed to parse
```

Comment:
nit:

```suggestion
    /// The given query failed to parse.
```

Reply:
Fixed in https://github.com/alexpovel/srgn/pull/144/commits/5c08ce08de67f30c77d691e4c0309efbe31f2bec

Reply:
I'll make a new PR that adds a config file for the [`typos`](https://github.com/crate-ci/typos) CLI tool if you don't have any objections?

Reply:
Yes feel free to go ahead! Love catching typos.